src/types/input.ts

```diff
@@ -0,0 +1,13 @@
+import { TooltipAttributes } from '@/types/label'
+
+export type Size = 'small' | 'medium' | 'large'
+export type SizeRecord = Record<Size, Size>
+
+export type IconPosition = 'start' | 'end'
+export type IconPositionRecord = Record<IconPosition, IconPosition>
+
+export interface LabelAttributes {
+  help: string;
+  info: string;
```

Comment:
Can you remove the semi-colons? (and search the other type files for semi-colons and commas and remove if not needed)?